scripts/cumberbatch.coffee

```diff
@@ -0,0 +1,19 @@
+# Description:
+#   Benedict Cumberbatch name generator.
+#
+# Dependencies:
+#   None
+#
+# Configuration:
+#   None
+#
+# Commands:
+#   cumberbatch - Creates your very own Benedict Cumberbatch
+#
+# Author:
+#   mbulmer
+
+module.exports = (robot) ->
+  robot.respond /(cumberbatch)/i, (msg) ->
```

Comment:
Similar scripts used robot.respond so I followed suit. Is robot.hear more appropriate for these?